tests/brain/python/compartmentReportReader.py

```diff
@@ -54,51 +55,80 @@ def test_mapping(self):
 
     def test_frames(self):
         view = self.report.create_view({1,2,3})
-        frames = view.load(0.0,0.2)
-        assert(len(frames) == 2)
-        frames = view.load(0.05,0.25)
+        timestamps ,frames = view.load(0.0,0.2)
+        assert(len(timestamps) == 2)
+        assert(frames.shape == (2,3))
+        timestamps ,frames = view.load(0.05,0.25)
         # This window overlaps frames [0, 0.1), [0.1, 0.2), [0.2, 03)
-        assert(len(frames) == 3)
-        frames = view.load(0.09,0.12)
+        assert(len(timestamps) == 3)
+        assert(frames.shape == (3,3))
+        timestamps ,frames = view.load(0.09,0.12)
         # Despite the time window is smaller than the timestep, each edge is
         # falling on a different frame, so we get two frames.
-        assert(len(frames) == 2)
-        frames = view.load_all()
-        assert(len(frames) == 100)
-        frame = view.load(0.1)
-        assert(frame.timestamp == 0.10000000149011612)
-        assert(frame.empty == False)
-        assert(frame.data.tolist() ==
+        assert(len(timestamps) == 2)
+        assert(frames.shape == (2,3))
+        timestamps ,frames = view.load_all()
+        assert(len(timestamps) == 100)
+        timestamp ,frame = view.load(0.1)
+        assert(timestamp == 0.1)
+        assert(frame.shape == (3,))
+        assert(frame.tolist() ==
                [-65.14350891113281, -65.29447937011719, -65.4448013305664])
 
 class TestMemoryManagement(unittest.TestCase):
 
     def test_view(self):
-        view = CompartmentReportReader(report_path).create_view({1, 2, 3})
+        view = CompartmentReport(report_path).create_view({1, 2, 3})
         assert(view.gids == {1, 2, 3})
 
     def test_frame(self):
-        frame = CompartmentReportReader(report_path).create_view(
+        timestamp, frame = CompartmentReport(report_path).create_view(
             {1, 2, 3}).load(0.1)
-        assert(frame.timestamp == 0.10000000149011612)
-        assert(frame.empty == False)
-        assert(frame.data.tolist() ==
+
+        assert(timestamp == 0.1)
+        assert(frame.tolist() ==
                [-65.14350891113281, -65.29447937011719, -65.4448013305664])
 
     def test_frame_data(self):
-        data = CompartmentReportReader(report_path).create_view(
-            {1, 2, 3}).load(0.1).data
+        time, data = CompartmentReport(report_path).create_view(
+            {1, 2, 3}).load(0.1)
         assert(data.tolist() ==
                [-65.14350891113281, -65.29447937011719, -65.4448013305664])
 
     def test_mapping(self):
 
-        mapping = CompartmentReportReader(report_path).create_view(
+        mapping = CompartmentReport(report_path).create_view(
             {1, 2, 3}).mapping
         assert(mapping.num_compartments(1) == 1)
         assert(mapping.index.tolist() ==
                [(0, 1, 0, 1), (1, 2, 0, 1), (2, 3, 0, 1)])
 
+
+class TestExample(unittest.TestCase):
```

Comment:
I think there's no need to add a test with the example code from the spec. Just verify that existing tests have the proper code coverage or add a new one if needed.